app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: int) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_ratings: int) -> None:
+        self.distance_center = distance_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_ratings
+
+    def calculate_washing_price(self, car_info: callable) -> float:
+        return round((car_info.comfort_class
+                      * (self.clean_power
+                         - car_info.clean_mark)
+                      * self.average_rating
+                      / self.distance_center)
+                     , 1)
+
+    def serve_cars(self, car_list: list) -> float:
+        price = 0
+        for cars in car_list:
```

Comment:
The variable name 'cars' in the for loop is misleading. It suggests that it's a collection of cars, but it actually represents a single car. Consider renaming it to 'car'.